crds/run_v1beta1_runjob.yaml

```diff
@@ -295,113 +281,121 @@ spec:
                                               - external
                                             properties:
                                               external:
-                                                description: 'Allowed value: The `version`
-                                                  field of a `SecretManagerSecretVersion`
-                                                  resource.'
+                                                description: ' If provided must be
+                                                  in the format `projects/*/secrets/*/versions/*`.'
                                                 type: string
                                               name:
-                                                description: 'Name of the referent.
-                                                  More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names'
+                                                description: The `name` field of a
+                                                  `SecretManagerSecretVersion` resource.
                                                 type: string
                                               namespace:
-                                                description: 'Namespace of the referent.
-                                                  More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/'
+                                                description: The `metadata.namespace`
+                                                  field of a `SecretManagerSecretVersion`
+                                                  resource.
                                                 type: string
                                             type: object
-                                        required:
```

Comment:
We don't support rolling back versions. However this represents a clear example of we cannot rollback 1.135 to anything earlier.